lib/rules/jsx-indent-props.js

```diff
@@ -150,14 +150,16 @@ module.exports = {
 
       const indent = regExp.exec(src);
       const useOperator = /^([ ]|[\t])*[:]/.test(src) || /^([ ]|[\t])*[?]/.test(src);
-      const useBracket = /^([ ]|[\t])*[<]/.test(src);
 
       line.currentOperator = false;
       if (useOperator) {
         line.isUsingOperator = true;
         line.currentOperator = true;
-      } else if (useBracket) {
-        line.isUsingOperator = false;
+      } else {
+        const useBracket = /^([ ]|[\t])*[<]/.test(src);
```

Comment:
can we make this regex at module level, rather than nesting it deeper? (same with the useOperator regexes)

Reply:
I'm relatively sure that's not unique to ES5, that's eternally been the semantics in JS.